docs/manual/pubsub.md

```diff
@@ -38,6 +38,8 @@ stream of messages where the first element indicates the type of
 message. The commands that are allowed in the context of a subscribed
 client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.
 
+In particular, the operation of the `UNSUBSCRIBE` and `PUNSUBSCRIBE` commands in redis-cli has limitations, so keep in mind.
```

Comment:
I think it would be clearer to combine this statement with the next statement regarding redis-cli pub/sub limitations.
maybe something like:
```
Please note that `redis-cli` will not accept any commands once in
subscribed mode including the allowed commands mentioned earlier.
In particular, you will not be able to use the `UNSUBSCRIBE` and `PUNSUBSCRIBE`  commands 
via redis-cli once in subscribed mode and can only quit the mode with `Ctrl-C`.
```

Reply:
@ranshid, Thank you for your review.
Yes, I think it's good to mention it in a combined sentence. :)